administration/src/util/translations/de.json

```diff
@@ -561,7 +562,9 @@
     "regionNotUnique": "Den Akzeptanzstellen konnte keine eindeutige Region zugewiesen werden. Bitte wenden Sie sich an den IT-Support",
     "pdfCreationError": "Etwas ist schiefgegangen beim Erstellen der PDF.",
     "csvCreationError": "Etwas ist schiefgegangen beim Erstellen der CSV.",
-    "freinetAgencyNotFound": "Diese Freinetagentur existiert nicht."
+    "freinetAgencyNotFound": "Diese Agentur konnte in Freinet nicht gefunden werden.",
+    "freinetMultiplePersonError": "Es wurden mehrere Personen mit diesen Daten gefunden. Bitte aktualisieren Sie die Daten in Freinet-Online manuell.",
```

Comment:
@connium i think we have to find here a better description like:
"Der Antragsteller konnte nicht eindeutig einer Person in Freinet-Online zugewiesen werden. Bitte aktualisieren Sie die Daten in Freinet-Online manuell."

Maybe you got even a better idea

Reply:
@f1sh1918 @hauf-toni how about "Der antragstellenden Person konnte keine Person in Freinet-Online zugeordnet werden. Bitte aktualisieren Sie die Daten in Freinet-Online manuell."